test/test.js

```diff
@@ -402,7 +404,7 @@ describe('FireWatch Compound Case', function () {
         this.timeout(10000);
         var tested = false;
 
-        var watcher = FireWatch.start( root, function () {
+        var watcher = FireWatch.start( root + '/', function () {
```

Comment:
Ok, i will add one when I get home

On Thu, Aug 6, 2015 at 5:10 PM, Wu Jie <notifications@github.com> wrote:

> In test/test.js
> <https://github.com/fireball-x/fire-watch/pull/1#discussion_r36394442>:
>
> > @@ -402,7 +404,7 @@ describe('FireWatch Compound Case', function () {
> >          this.timeout(10000);
> >          var tested = false;
> >
> > -        var watcher = FireWatch.start( root, function () {
> > +        var watcher = FireWatch.start( root + '/', function () {
>
> it would be better to add a single case for it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fireball-x/fire-watch/pull/1/files#r36394442>.
>



-- 
Yorkie Liu
https://github.com/yorkie
+18600219033